src/Microsoft.Azure.ServiceBus/Resources.resx

```diff
@@ -159,6 +159,9 @@
   <data name="TokenMissingExpiresOn" xml:space="preserve">
     <value>The provided token does not specify the 'ExpiresOn' value.</value>
   </data>
+  <data name="ListOfLockTokensCannotBeEmty" xml:space="preserve">
```

Comment:
nit - please fix typo ListOfLockTokensCannotBeEmpty in the next commit

Reply:
@axisc I'm blind as  a 🦇 - what typo?

Reply:
ListOfLockTokensCannotBeEmty > ListOfLockTokensCannotBeEmpty

Reply:
Thanks @stjepan. Blind like a bat I am 😂

Reply:
PR #583 raised